app/controllers/companies_controller.rb

```diff
@@ -1,16 +1,40 @@
 class CompaniesController < ApplicationController
   before_action :find_company, only: [:show, :edit, :update, :destroy]
-  before_action :pre_form, only: [:new, :edit]
+  before_action :pre_form, only: [:new, :edit, :create]
```

Comment:
To co tam https://github.com/Offmat/puzzle_room/pull/5/files#diff-ee1121aa515987658d9c9a38de73e802R3